src/world.c

```diff
@@ -1858,3 +1858,26 @@ void StartFrame(int time)
 
 	TeamplayGameTick();
 }
+
+// Check the same spawnflags as items only visible in DM for monsters as well.
+// 1. Disabled for all SP modes
+// 2. Not disabled for DM
+qbool AllowMonster(gedict_t *e)
```

Comment:
Decent place?

Reply:
sure!

And sorry for not responding earlier. I completely overlooked/forgot about this PR.

Just keep reminding me next time ;)